WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewViewModel.swift

```diff
@@ -170,6 +213,29 @@ final class ProductDetailPreviewViewModel: ObservableObject {
         shouldShowFeedbackView = false
     }
 
+    // MARK: Switch options
+    func next() {
```

Comment:
Nit: this method name could be improved for clarity:

```suggestion
    func switchToNextOption() {
```

Reply:
Done in 163c44fb3ac21a2db3147551b850e0d4de2221a7